batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/services/UserService.java

```diff
@@ -3,15 +3,34 @@
 import com.lftechnology.batch7crud.dao.UserDao;
 import com.lftechnology.batch7crud.exception.DataException;
 import com.lftechnology.batch7crud.model.User;
+import com.lftechnology.batch7crud.util.PasswordGenerator;
+
+import java.security.NoSuchAlgorithmException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 /**
  * Created by Grishma Shrestha grishmashrestha@lftechnology.com on 1/18/16.
  */
 public class UserService {
   UserDao userDao = new UserDao();
+  private static final Logger LOGGER = Logger.getLogger(UserService.class.getName());
 
   public User create(User user) throws DataException {
     return userDao.create(user);
   }
 
+  public User fetchUserByUsernameAndPassword (User user) throws DataException {
+    String passwordHash;
+    try {
+      passwordHash = PasswordGenerator.hashUsingMD5(user.getPassword());
+      user.setPassword(passwordHash);
+      System.out.println(passwordHash);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)